src/ell/studio/config.py

```diff
@@ -10,31 +11,36 @@
 
 # todo. maybe we default storage dir and other things in the future to a well-known location
 # like ~/.ell or something
-@lru_cache
+@lru_cache(maxsize=1)
 def ell_home() -> str:
     return os.path.join(os.path.expanduser("~"), ".ell")
 
 
 class Config(BaseModel):
     pg_connection_string: Optional[str] = None
     storage_dir: Optional[str] = None
-
-    @classmethod
-    def create(
-        cls,
-        storage_dir: Optional[str] = None,
-        pg_connection_string: Optional[str] = None,
-    ) -> 'Config':
-        pg_connection_string = pg_connection_string or os.getenv("ELL_PG_CONNECTION_STRING")
-        storage_dir = storage_dir or os.getenv("ELL_STORAGE_DIR")
-
+    mqtt_connection_string: Optional[str] = None
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def model_post_init(self, __context):
```

Comment:
This should let us do Config() instead of Config.create() and have the same functionality